src/panels/config/entities/dialog-entity-editor.ts

```diff
@@ -3,6 +3,7 @@ import "@polymer/paper-dialog-scrollable/paper-dialog-scrollable";
 import "../../../components/ha-icon-button";
 import "@polymer/paper-tabs/paper-tab";
 import "@polymer/paper-tabs/paper-tabs";
+import "./editor-tabs/settings/entity-settings-helper-tab";
```

Comment:
This should be imported dynamically, see <https://github.com/home-assistant/frontend/pull/5994/files#diff-8ef4548c9ea44d5a02deb2d69167cf00R222>